manim/scene/vector_space_scene.py

```diff
@@ -558,7 +613,6 @@ def setup(self):
         self.transformable_labels = []
         self.moving_mobjects = []
 
-        self.t_matrix = np.array(self.t_matrix)
```

Comment:
Is this not used anywhere?

Reply:
Nope, it was defined arbitrarily. Not relied on anywhere else.